packages/shared_preferences/shared_preferences_tool/test/src/shared_preferences_state_notifier_test.mocks.dart

```diff
@@ -0,0 +1,103 @@
+// Mocks generated by Mockito 5.4.4 from annotations
```

Comment:
@stuartmorgan is there a policy to follow with respect to checking in generated mock files?

Reply:
@stuartmorgan this is the only open question left that I have on this PR.

Reply:
Sorry, I missed this question before. It's fine to check in generated mocks.